osgp/protocol-adapter-oslp/web-device-simulator/src/main/java/org/opensmartgridplatform/webdevicesimulator/service/OslpChannelHandler.java

```diff
@@ -1122,6 +1039,25 @@ private void handleSetTransitionRequest(final Device device, final SetTransition
         }
     }
 
+    private void sendEvent(final Device device, final Oslp.Event event, final String description) {
+        this.sendEventWithCustomDelay(device, event, description, 3000);
```

Comment:
Similar as with the device registration delay. If we can already imagine a realistic chance that variation across environments of the specific delay will come in handy, it could be better to make it configurable.
If this is deemed OK in different use cases, I think it might be good enough as it is.

Reply:
This is an existing delay, not updating or moving it to properties for now.